experimental/dds/tree2/src/test/feature-libraries/editable-tree-2/proxies.spec.ts

```diff
@@ -114,6 +114,6 @@ describe("SharedTreeObject", () => {
 function createTypedTreeView<TRoot extends FieldSchema>(
```

Comment:
You can probably remove this function now, I don't think it does anything anymore.